src/libs/AxelarQueryAPI.ts

```diff
@@ -22,6 +22,22 @@ interface TranslatedTransferRateLimitResponse {
   outgoing: string;
   limit: string;
 }
+interface GMPExpressTransactionParameters {
+  isGMPExpressTransaction: boolean;
+  transferAmount: BigNumber;
+  destinationContractAddress: `0x${string}`;
+  evmTokenSymbol: string;
+}
+export interface AxelarQueryAPIFeeResponse {
+  fee: string;
+  expressFee: string;
+  baseFee: string;
+  dynamicFee: string;
```

Comment:
```suggestion
  executionFee: string;
```
Other fees are also dynamic to gas prices now